airgun/session.py

```diff
@@ -421,6 +422,11 @@ def package(self):
         """Instance of Packge entity."""
         return PackageEntity(self.browser)
 
+    @cached_property
+    def modulestream(self):
+        """Instance of Packge entity."""
```

Comment:
Obviously it's ModuleStream entity, not Package. (And maybe fix typo in Package comment, while you're at it?)

Reply:
yeah that is a typo issue

Reply:
Done!